lib/customer-deposit-wallet/http/Cardano/Wallet/Deposit/HTTP/Types/JSON.hs

```diff
@@ -140,3 +147,27 @@ instance ToSchema (ApiT CustomerList) where
             $ NamedSchema
                 (Just "ApiT CustomerList")
                 customerListSchema
+
+instance ToJSON (ApiT ChainPoint) where
+    toJSON (ApiT Origin) = "origin"
+    toJSON (ApiT (At sl)) = object
+        [ "at_slot" .= toJSON sl
+        ]
```

Comment:
```suggestion
    toJSON (ApiT Origin) = "genesis"
    toJSON (ApiT (At sl)) = object
        [ "slot_no" .= toJSON sl
        ]
```

The current state of the nomenclature for the number is this:

* `ouroboros-consensus` and `cardano-ledger-core` have settled on `SlotNo`
* The ledger specification calls it `Slot`.

The `ChainPoint` type is transitional architecture, I want to import it from `Cardano.Wallet.Read` instead, specifically here:

https://github.com/cardano-foundation/cardano-wallet-agda/blob/433c08fbdce295cfb8206f47b1a5f19196f6e248/lib/cardano-wallet-read/haskell/Cardano/Wallet/Read/Chain.hs#L55-L61

This suggests the above naming conventions for the fields.

Reply:
:+1: